packages/front-end/components/SavedGroups/LargeSavedGroupSupportWarning.tsx

```diff
@@ -0,0 +1,85 @@
+import { getConnectionSDKCapabilities } from "shared/sdk-versioning";
+import { SDKConnectionInterface } from "@back-end/types/sdk-connection";
+import Link from "next/link";
+import { SMALL_GROUP_SIZE_LIMIT } from "shared/util";
+import React from "react";
+import { PiInfoFill } from "react-icons/pi";
+import useSDKConnections from "@/hooks/useSDKConnections";
+
+export function useLargeSavedGroupSupport(
+  project?: string
+): {
+  supportedConnections: SDKConnectionInterface[];
+  unsupportedConnections: SDKConnectionInterface[];
+  unversionedConnections: SDKConnectionInterface[];
+} {
+  const { data: sdkConnectionData } = useSDKConnections();
+  let connections = sdkConnectionData?.connections || [];
+  connections = connections.filter(
+    // TODO: check that projects array being empty does include all projects
+    (conn) =>
+      conn.projects.length === 0 || conn.projects.includes(project || "")
+  );
```

Comment:
@bryce-fitzsimons need to double check this with you

Reply:
try to use `getConnectionsSDKCapabilities`, append if necessary